File: ru.kmorozov.gbd.core/src/main/java/ru/kmorozov/gbd/utils/ClassUtils.java
package ru.kmorozov.gbd.utils;

/**
 * Created by km on 23.12.2016.
 */
public class ClassUtils {

    public static boolean isClassExists(String className) {
        try {
            Class<?> clazz = Class.forName(className);
            return null != clazz;
        } catch (ClassNotFoundException e) {
            return false;
        }
    }
}

File: ru.kmorozov.gbd.core/src/main/java/ru/kmorozov/gbd/utils/ClassUtils.kt
package ru.kmorozov.gbd.utils

/**
 * Created by km on 23.12.2016.
 */
object ClassUtils {

    fun isClassExists(className: String): Boolean {
        try {
            val clazz = Class.forName(className)
            return null != clazz
        } catch (e: ClassNotFoundException) {
            return false
        }

    }
}
